h2o-r/h2o-package/R/communication.R

```diff
@@ -849,9 +849,17 @@ h2o.show_progress <- function() assign("PROGRESS_BAR", TRUE, .pkg.env)
       if (keepRunning) {
         Sys.sleep(pollInterval)
         if(verboseModelScoringHistory){
-          cat(paste0("\nScoring History for Model ",job$dest$name, " at ", Sys.time(),"\n"))
-          print(paste0("Model Build is ", job$progress*100, "% done..."))
-          print(tail(h2o.getModel(job$dest$name)@model$scoring_history))
+          tryCatch({
+            cat(paste0("\nScoring History for Model ",job$dest$name, " at ", Sys.time(),"\n"))
+            print(paste0("Model Build is ", job$progress*100, "% done..."))
+          }, error = function(e){
+            print("Model build is starting now...") #Catch 404 with model
```

Comment:
How can there be 404 error? I don't see a HTTP call in this block

Reply:
The 404 comes from the REST call, which is handled by `h2o.doSafeRest`

Reply:
But there is no such call in this block

```
            cat(paste0("\nScoring History for Model ",job$dest$name, " at ", Sys.time(),"\n"))
            print(paste0("Model Build is ", job$progress*100, "% done..."))
```

Or is it?

Reply:
Jumped the gun there. it can be removed.

Reply:
:)